nav2_gps_waypoint_follower_demo/worlds/city_world/city_world.model

```diff
@@ -0,0 +1,371 @@
+<?xml version="1.0" ?>
```

Comment:
This should be one of the only files you should need to include

Reply:
yes if all of the models included were available in [gazebo_models](https://github.com/osrf/gazebo_models), again majority of models are available in `gazebo_models` but not all of them.